rafay/rafay_cluster_override_sharing.go

```diff
@@ -0,0 +1,516 @@
+package rafay
```

Comment:
This file `rafay_cluster_override_sharing.go` may be not needed. we can delete right..?